jenkins/osa_onmetal.functions.groovy

```diff
@@ -316,9 +316,7 @@ def stop_during_test() {
 }
 
 
-def aggregate_results() {
-
-    String host_ip = get_onmetal_ip()
+def aggregate_results(host_ip) {
```

Comment:
You also need to modify the pipeline file to send this parameter or the pipeline will break since it is a required argument.